source/code/plugins/filter_syslog.rb

```diff
@@ -30,6 +30,7 @@ def filter(tag, time, record)
       # Use Time.now, because it is the only way to get subsecond precision in version 0.12.
       # The time may be slightly in the future from the ingestion time.
       record["Timestamp"] = OMS::Common.format_time(Time.now.to_f)
+      record["EventTime"] = OMS::Common.format_time(time)
```

Comment:
Fluentd supports syslog forwarding, so I would think that the fluentd config or syslog config would have to handle that